tests/L0/run_optimizers/test_lamb.py

```diff
@@ -190,7 +196,7 @@ def get_max_diff(self, ref_param, tst_param):
 
         return max_abs_diff, max_rel_diff
 
-    def gen_single_type_test(self, param_type=torch.float, device="cuda"):
+    def gen_single_type_test(self, param_type=torch.float, device="cuda", iters=None):
```

Comment:
see the comment on `test_bfloat16`

```suggestion
    def gen_single_type_test(self, param_type=torch.float, device="cuda"):
```